Tests/PSGetInstallModule.Tests.ps1

```diff
@@ -23,6 +23,16 @@ function SuiteSetup {
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:TempPath = Get-TempPath
     $null = New-Item -Path $script:MyDocumentsModulesPath -ItemType Directory -ErrorAction SilentlyContinue -WarningAction SilentlyContinue
+    
```

Comment:
Consider moving all the prerelease feature specific tests into a single pester tests.ps1 file.